libs/external-db-postgres/src/connection_provider.ts

```diff
@@ -32,7 +32,12 @@ const init = (cfg, _poolOptions) => {
     const dataProvider = new DataProvider(pool, filterParser)
     const schemaProvider = new SchemaProvider(pool)
 
-    return { dataProvider: dataProvider, schemaProvider: schemaProvider, databaseOperations, connection: pool, cleanup: async() => await pool.end(() => { }) }
+    return { 
+        dataProvider: dataProvider,
+        schemaProvider: schemaProvider,
+        databaseOperations, 
+        connection: pool, 
+        cleanup: async() => pool.end(() => {}) 
```

Comment:
why you removed the await?

Reply:
![image](https://user-images.githubusercontent.com/407079/175807557-6efa7a5a-1b0c-4931-9c74-b0fb9a6b5c18.png)